docs/kotlin.core/overload-resolution.md

```diff
@@ -218,7 +218,7 @@ A function is *applicable* for a specific call if and only if the function param
 
 #### Description
 
-Determining function applicability for a specific call is a [type constraint][Type constraints] problem.
+Determining function applicability for a specific call is a [type constraint][Kotlin Type constraints] problem.
```

Comment:
To better follow the capitalization rules, this should be `Kotlin type constraints`